e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -191,16 +192,15 @@ test.describe('Example Imagery Object', () => {
         expect.soft(zoomedInBoundingBox.height).toBeGreaterThan(initialBoundingBox.height);
         expect.soft(zoomedInBoundingBox.width).toBeGreaterThan(initialBoundingBox.width);
 
-        await zoomResetBtn.click();
         // wait for zoom animation to finish
-        await page.locator(backgroundImageSelector).hover({trial: true});
-
-        const resetBoundingBox = await page.locator(backgroundImageSelector).boundingBox();
-        expect.soft(resetBoundingBox.height).toBeLessThan(zoomedInBoundingBox.height);
-        expect.soft(resetBoundingBox.width).toBeLessThan(zoomedInBoundingBox.width);
+        await expect.poll(async () => {
```

Comment:
I think we have a legitimate bug here in the application. Let's file an issue so that we don't forget about this and come back to it soon. Just leave it as a comment right here about zoom flaking out